### PR TITLE
Build with dotnet

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Please review @rynowak @pranavkm

### CODE REVIEW

**Comment by pranavkm** [comment]
:shipit:

File: build.cmd [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-    CALL packages\KoreBuild\build\dnvm install %BUILDCMD_DNX_VERSION% -runtime CoreCLR -arch x86 -alias default
-    CALL packages\KoreBuild\build\dnvm install default -runtime CLR -arch x86 -alias default
-) ELSE (
-    CALL packages\KoreBuild\build\dnvm use default -runtime CLR -arch x86
+
+IF NOT EXIST %KOREBUILD_FOLDER% (
+    SET KOREBUILD_DOWNLOAD_ARGS=
+    IF NOT "%KOREBUILD_VERSION%"=="" (
+        SET KOREBUILD_DOWNLOAD_ARGS=-version %KOREBUILD_VERSION%
+    )
```
Comment (dougbu) [inline build.cmd]: Move this `IF` above `IF NOT EXIST %KOREBUILD_FOLDER%`

File: test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json [hunk 2]
```diff
@@ -2,15 +2,27 @@
...
+  "compilationOptions": {"define": ["TEST"]},
+  "testRunner": "xunit",
+   "commands": {
+     "test": "xunit.runner.aspnet"
+  },
+  "frameworks": {
+    "dnx451": {
+      "frameworkAssemblies": {
+        "System.Runtime": "",
+        "System.Threading.Tasks": ""
```
Comment (dougbu) [inline test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json]: Why?

**Comment by dougbu** [comment]
:watch:

**Comment by victorhurdugaci** [comment]
Updated

**Comment by dougbu** [comment]
:shipit:

**Pull request merged**